src/chainparams.cpp

```diff
@@ -163,13 +163,13 @@ class CTestNetParams : public CMainParams {
         nMaxTipAge = 0x7fffffff;
         nPruneAfterHeight = 1000;
 
-        // 1MB max blocks before 1 Aug 2015
-        // Then, if miner consensus: 8MB max, doubling every two years
+        // 1MB max blocks before 1 Mar 2014
+        // Then, if miner consensus: 2MB max, doubling twice over two years
         consensus.nMaxSizePreFork = 1000*1000; // 1MB max pre-fork
-        consensus.nEarliestSizeForkTime = 1438387200; // 1 Aug 2015 00:00:00 UTC
+        consensus.nEarliestSizeForkTime = 1393632000; // 01 Mar 2014 00:00:00 UTC, 2 years ahead of mainnet
```

Comment:
So if it takes until September before we get miner consensus, that means we fork straight into 2.5 MB blocks? Is that what miners want?